sdk/servicebus/service-bus/src/dataTransformer.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { message } from "rhea-promise";
+import isBuffer from "is-buffer";
+import { Buffer } from "buffer";
+import { logErrorStackTrace, logger } from "./log";
+
+/**
+ * Describes the transformations that can be performed to encode/decode the data before sending it
+ * on (or receiving it from) the wire.
+ * @internal
+ * @ingore
+ */
+export interface DataTransformer {
+  /**
+   * @property {Function} encode A function that takes the body property from an EventData object
```

Comment:
```suggestion
   * @property {Function} encode A function that takes the body property from a ServiceBusMessage object
```

Reply:
Oh my, that's been there a while, nice catch!